src/tests/multihost/alltests/test_default_debug_level.py

```diff
@@ -156,28 +156,55 @@ def test_0004_bz1893159(self, multihost, backupsssdconf):
         """
         :title: default debug logs: Check default level 2
         :id: d44d5883-fc52-418d-b407-3ac63f7104d8
+        :setup:
+          1. Remove debug_level option from sssd.conf
+          2. Set domains = typo_domain (non existing) in [sssd] section
         :steps:
-          1. Remove debug_level from sssd.conf
-          2. Start sssd after clearing cache and logs
-          3. Kill pid of sssd with signal SIGUSR2
-          4. Check logs
+          1. Start sssd after clearing cache and logs
+          2. Check sssd.log contains log related to 'SSSD cannot load config'
         :expectedresults:
-          1. sssd should use default debug level with no level defined
-          2. Succeeds
-          3. sssd process is killed
-          4. logs of level of 0x0040 are in the log file
+          1. SSSD will not start
```

Comment:
This is not checked.
result cmd from L176 should be stored and assert on retirncode should be added.

alternatively L176 can be inside "with pytest.raises(Exception) as e_info:" checking that the proper exception is raised.

Reply:
moved the sssd start part to setup. Goal here is to invoke the error condition in sssd process, so a log in related logfile could be parsed.